ui/app/templates/vault/cluster/secrets/backend/overview.hbs

```diff
@@ -32,26 +32,31 @@
   <div class="box is-fullwidth is-shadowless has-tall-padding">
     <div class="selectable-card-container {{if (and (eq this.model.connections 403) (eq this.model.roles 403)) 'one-card'}}">
       {{#if this.model.connectionCapabilities.canList}}
-        <SelectableCard
+        <OverviewCard
+          data-test-selectable-card="Connections"
```

Comment:
Do we want to update the `data-test` to be a different name since these aren't selectable cards anymore?

Reply:
I can do that